src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -12,37 +12,59 @@ public class DotnetFirstTimeUseConfigurer
         private IEnvironmentProvider _environmentProvider;
         private INuGetCachePrimer _nugetCachePrimer;
         private INuGetCacheSentinel _nugetCacheSentinel;
+        private IFirstTimeUseNoticeSentinel _firstTimeUseNoticeSentinel;
 
         public DotnetFirstTimeUseConfigurer(
             INuGetCachePrimer nugetCachePrimer,
             INuGetCacheSentinel nugetCacheSentinel,
+            IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel,
             IEnvironmentProvider environmentProvider)
         {
             _nugetCachePrimer = nugetCachePrimer;
             _nugetCacheSentinel = nugetCacheSentinel;
+            _firstTimeUseNoticeSentinel = firstTimeUseNoticeSentinel;
             _environmentProvider = environmentProvider;
         }
 
         public void Configure()
         {
-            if(ShouldPrimeNugetCache())
+            if (ShouldPrintFirstTimeUseNotice())
             {
                 PrintFirstTimeUseNotice();
+            }
+
+            if (ShouldPrimeNugetCache())
+            {
+                string cachePrimeMessage = LocalizableStrings.NugetCachePrimeMessage;
+                Reporter.Output.WriteLine();
+                Reporter.Output.WriteLine(cachePrimeMessage);
 
                 _nugetCachePrimer.PrimeCache();
             }
         }
 
+        private bool ShouldPrintFirstTimeUseNotice()
+        {
+            return ShouldRunFirstRunExperience() && !_firstTimeUseNoticeSentinel.Exists();
```

Comment:
What do you think about calling it `DOTNET_PRINT_TELEMETRY_MESSAGE` that defaults to `true`?